[2025-01-23T13:04:16.981+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: task45_train_models_v2.task1_repareData manual__2025-01-23T13:04:12.413212+00:00 [queued]>
[2025-01-23T13:04:16.997+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: task45_train_models_v2.task1_repareData manual__2025-01-23T13:04:12.413212+00:00 [queued]>
[2025-01-23T13:04:16.998+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2025-01-23T13:04:17.035+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): task1_repareData> on 2025-01-23 13:04:12.413212+00:00
[2025-01-23T13:04:17.048+0000] {standard_task_runner.py:60} INFO - Started process 7020 to run task
[2025-01-23T13:04:17.057+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'task45_train_models_v2', 'task1_repareData', 'manual__2025-01-23T13:04:12.413212+00:00', '--job-id', '84', '--raw', '--subdir', 'DAGS_FOLDER/task45_train_models_v2.py', '--cfg-path', '/tmp/tmpf36o3098']
[2025-01-23T13:04:17.059+0000] {standard_task_runner.py:88} INFO - Job 84: Subtask task1_repareData
[2025-01-23T13:04:17.082+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:194: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2025-01-23T13:04:17.162+0000] {task_command.py:423} INFO - Running <TaskInstance: task45_train_models_v2.task1_repareData manual__2025-01-23T13:04:12.413212+00:00 [running]> on host a8cdec4ae828
[2025-01-23T13:04:17.452+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='task45_train_models_v2' AIRFLOW_CTX_TASK_ID='task1_repareData' AIRFLOW_CTX_EXECUTION_DATE='2025-01-23T13:04:12.413212+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-23T13:04:12.413212+00:00'
[2025-01-23T13:04:17.493+0000] {warnings.py:109} WARNING - /opt/***/dags/task45_train_models_v2.py:29: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  df_temp.loc[:, 'target'] = df_temp['temperature'].shift(1)

[2025-01-23T13:04:17.501+0000] {warnings.py:109} WARNING - /opt/***/dags/task45_train_models_v2.py:33: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  df_temp.loc[:, 'temp_m-{}'.format(i)

[2025-01-23T13:04:17.506+0000] {warnings.py:109} WARNING - /opt/***/dags/task45_train_models_v2.py:33: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  df_temp.loc[:, 'temp_m-{}'.format(i)

[2025-01-23T13:04:17.514+0000] {warnings.py:109} WARNING - /opt/***/dags/task45_train_models_v2.py:33: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  df_temp.loc[:, 'temp_m-{}'.format(i)

[2025-01-23T13:04:17.516+0000] {warnings.py:109} WARNING - /opt/***/dags/task45_train_models_v2.py:33: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  df_temp.loc[:, 'temp_m-{}'.format(i)

[2025-01-23T13:04:17.521+0000] {warnings.py:109} WARNING - /opt/***/dags/task45_train_models_v2.py:33: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  df_temp.loc[:, 'temp_m-{}'.format(i)

[2025-01-23T13:04:17.531+0000] {warnings.py:109} WARNING - /opt/***/dags/task45_train_models_v2.py:33: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  df_temp.loc[:, 'temp_m-{}'.format(i)

[2025-01-23T13:04:17.539+0000] {warnings.py:109} WARNING - /opt/***/dags/task45_train_models_v2.py:33: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  df_temp.loc[:, 'temp_m-{}'.format(i)

[2025-01-23T13:04:17.546+0000] {warnings.py:109} WARNING - /opt/***/dags/task45_train_models_v2.py:33: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  df_temp.loc[:, 'temp_m-{}'.format(i)

[2025-01-23T13:04:17.552+0000] {warnings.py:109} WARNING - /opt/***/dags/task45_train_models_v2.py:33: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  df_temp.loc[:, 'temp_m-{}'.format(i)

[2025-01-23T13:04:17.559+0000] {warnings.py:109} WARNING - /opt/***/dags/task45_train_models_v2.py:29: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  df_temp.loc[:, 'target'] = df_temp['temperature'].shift(1)

[2025-01-23T13:04:17.570+0000] {warnings.py:109} WARNING - /opt/***/dags/task45_train_models_v2.py:33: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  df_temp.loc[:, 'temp_m-{}'.format(i)

[2025-01-23T13:04:17.572+0000] {warnings.py:109} WARNING - /opt/***/dags/task45_train_models_v2.py:33: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  df_temp.loc[:, 'temp_m-{}'.format(i)

[2025-01-23T13:04:17.580+0000] {warnings.py:109} WARNING - /opt/***/dags/task45_train_models_v2.py:33: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  df_temp.loc[:, 'temp_m-{}'.format(i)

[2025-01-23T13:04:17.582+0000] {warnings.py:109} WARNING - /opt/***/dags/task45_train_models_v2.py:33: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  df_temp.loc[:, 'temp_m-{}'.format(i)

[2025-01-23T13:04:17.585+0000] {warnings.py:109} WARNING - /opt/***/dags/task45_train_models_v2.py:33: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  df_temp.loc[:, 'temp_m-{}'.format(i)

[2025-01-23T13:04:17.593+0000] {warnings.py:109} WARNING - /opt/***/dags/task45_train_models_v2.py:33: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  df_temp.loc[:, 'temp_m-{}'.format(i)

[2025-01-23T13:04:17.600+0000] {warnings.py:109} WARNING - /opt/***/dags/task45_train_models_v2.py:33: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  df_temp.loc[:, 'temp_m-{}'.format(i)

[2025-01-23T13:04:17.602+0000] {warnings.py:109} WARNING - /opt/***/dags/task45_train_models_v2.py:33: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  df_temp.loc[:, 'temp_m-{}'.format(i)

[2025-01-23T13:04:17.604+0000] {warnings.py:109} WARNING - /opt/***/dags/task45_train_models_v2.py:33: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  df_temp.loc[:, 'temp_m-{}'.format(i)

[2025-01-23T13:04:17.616+0000] {warnings.py:109} WARNING - /opt/***/dags/task45_train_models_v2.py:29: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  df_temp.loc[:, 'target'] = df_temp['temperature'].shift(1)

[2025-01-23T13:04:17.620+0000] {warnings.py:109} WARNING - /opt/***/dags/task45_train_models_v2.py:33: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  df_temp.loc[:, 'temp_m-{}'.format(i)

[2025-01-23T13:04:17.628+0000] {warnings.py:109} WARNING - /opt/***/dags/task45_train_models_v2.py:33: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  df_temp.loc[:, 'temp_m-{}'.format(i)

[2025-01-23T13:04:17.635+0000] {warnings.py:109} WARNING - /opt/***/dags/task45_train_models_v2.py:33: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  df_temp.loc[:, 'temp_m-{}'.format(i)

[2025-01-23T13:04:17.638+0000] {warnings.py:109} WARNING - /opt/***/dags/task45_train_models_v2.py:33: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  df_temp.loc[:, 'temp_m-{}'.format(i)

[2025-01-23T13:04:17.648+0000] {warnings.py:109} WARNING - /opt/***/dags/task45_train_models_v2.py:33: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  df_temp.loc[:, 'temp_m-{}'.format(i)

[2025-01-23T13:04:17.650+0000] {warnings.py:109} WARNING - /opt/***/dags/task45_train_models_v2.py:33: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  df_temp.loc[:, 'temp_m-{}'.format(i)

[2025-01-23T13:04:17.654+0000] {warnings.py:109} WARNING - /opt/***/dags/task45_train_models_v2.py:33: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  df_temp.loc[:, 'temp_m-{}'.format(i)

[2025-01-23T13:04:17.656+0000] {warnings.py:109} WARNING - /opt/***/dags/task45_train_models_v2.py:33: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  df_temp.loc[:, 'temp_m-{}'.format(i)

[2025-01-23T13:04:17.657+0000] {warnings.py:109} WARNING - /opt/***/dags/task45_train_models_v2.py:33: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  df_temp.loc[:, 'temp_m-{}'.format(i)

[2025-01-23T13:04:17.756+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-01-23T13:04:17.777+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=task45_train_models_v2, task_id=task1_repareData, execution_date=20250123T130412, start_date=20250123T130416, end_date=20250123T130417
[2025-01-23T13:04:17.853+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-01-23T13:04:17.915+0000] {taskinstance.py:3280} INFO - 3 downstream tasks scheduled from follow-on schedule check
